nixos/tests/php/unit.nix

```diff
@@ -0,0 +1,101 @@
+import ../make-test-python.nix ({pkgs, ...}:
+ let
+    testdir = pkgs.writeTextDir "www/info.php" ''
+      <?php
+        phpinfo();
+      ?>
+    '';
```

Comment:
Can be compressed to a single line, closing tags aren't needed and is even discouraged to use according to commonly used code styles.
```suggestion
    testdir = pkgs.writeTextDir "www/info.php" "<?php phpinfo();";
```

Reply:
Fixed, thanks.